modules/nf-core/amulety/antiberty/tests/nextflow.config

```diff
@@ -1,3 +1,3 @@
 process {
-    cpus = 1 // Needed for deterministic results
```

Comment:
This seems like a shame to loose the comments herel

Reply:
I think this is a current limitation in the formatter, it can't handle inline comments like this yet.

Reply:
That's the point of pre-commit though 😞